README.md

```diff
@@ -60,29 +87,27 @@ If the user answers "Allow once", then calling `RequestPermission` again during
 
 Once permission is requested, you can use the various methods and extension methods of IGeolocatorService.
 
-## Features
-
 ### Request the user's location
 
 Once you have obtained the location permission, it's a simple matter to obtain the location:
 
+``` cs
+if (isGranted){
+    location = await locationService.GetLocation(CancellationToken.None);
+}
+//or
+var locationOrDefault = await locationService.GetLocationOrDefault(CancellationToken.None);
 ```
-var location = await locationService.GetLocation(ct);
-```
-
-If the user has not granted the location permission, an `InvalidOperationException` is thrown; therefore, we recommend not calling this method if you know the user has denied the location permission and this is a normal scenario. If the app has not kept track of this information, we recommend calling this extension method instead:
 
-```
-var locationOrDefault = await locationService.GetLocationOrDefault(ct);
-```
+when using `Getlocation` if the user has not granted the location permission, an `InvalidOperationException` is thrown; therefore, we recommend not calling this method if you know the user has denied the location permission and this is a normal scenario. 
 
-This method will return null instead of throwing an exception, in case the permission was denied.
+In cases where the app has not maintained this information, we suggest employing the extension method named `GetLocationOrDefault` instead. This method will return null instead of throwing an exception, in case the permission was denied.
```

Comment:
"maintained" might not be the best word here. And whether the app persists the result of `RequestPermission` or not, wouldn't you also get issues if the permission gets removed (like via native settings)?